.github/workflows/ci.yml

```diff
@@ -67,19 +63,68 @@ jobs:
       - name: Test package
         run: python -m pytest -vv -rs -Wd
 
+  checks_alt:
+    name: Check Python ${{ matrix.python-version }}, ${{ matrix.arch }} on ${{ matrix.runs-on }}
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.11"]
+        runs-on: [ubuntu-latest]
+        arch: [aarch64]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - uses: uraimo/run-on-arch-action@v2.5.0
+        name: Install and test in image
+        with:
+          arch: ${{ matrix.arch }}
+          distro: ubuntu_latest
+          # Not required, but speeds up builds
+          githubToken: ${{ github.token }}
+          dockerRunArgs: |
+            --volume "${PWD}:/fastjet" --platform linux/arm64
+          install: |
+            uname -m
+            apt-get update
+            apt-get install -y ca-certificates gnupg2 build-essential patch
```

Comment:
```suggestion
            apt-get install -y ca-certificates gnupg2 python3-dev build-essential patch
```

> Noooooo

As you're hitting

```pytb
...
    swig_wrap.cpp:178:11: fatal error: Python.h: No such file or directory
      178 | # include <Python.h>
...
```

I assume this means that you need to add `python3-dev` so that the necessary header files are available.

When I successfully built this across both `amd64` and `arm64` earlier in addition to the base environment I had for other software I was building inside of `python:3.9-slim-bullseye`

```dockerfile
RUN apt-get -qq update && \
    apt-get -qq -y install --no-install-recommends \
      gcc \
      g++ \
      gfortran \
      make \
      cmake \
      vim \
      zlib1g \
      zlib1g-dev \
      libbz2-dev \
      rsync \
      bash-completion \
      python3-dev \
      wget \
      ghostscript \
      bc \
      git && \
    apt-get -y clean && \
    apt-get -y autoremove && \
    rm -rf /var/lib/apt/lists/*
```

I also had to add the following

```dockerfile
RUN apt-get -qq update && \
    apt-get -qq -y install --no-install-recommends \
      libboost-dev \
      libmpfr-dev \
      libgmp-dev \
      swig \
      autoconf \
      patch \
      automake \
      libtool && \
    apt-get -y clean && \
    apt-get -y autoremove && \
    rm -rf /var/lib/apt/lists/*
```

so you might need a subset of that. :/